keras_hub/src/models/gemma3/gemma3_vit.py

```diff
@@ -0,0 +1,304 @@
+import keras
+from keras import ops
+
+from keras_hub.src.models.pali_gemma.pali_gemma_vit import (
+    MultiHeadAttentionPooling,
+)
+from keras_hub.src.models.pali_gemma.pali_gemma_vit import (
+    PaliGemmaVitEmbeddings,
```

Comment:
We could consider forking here too. Cross cutting dependencies to pali gemma might be awkward for both. `keras_hub.layers` for general stuff we can really reuse across many models.